src/com/itschool/session5/homework/IncrementDecrement.java

```diff
@@ -0,0 +1,65 @@
+package com.itschool.session5.homework;
+
+public class IncrementDecrement {
+
+    public static void main(String[] args) {
+
+        System.out.println("Numarul pana la 10: ");
+
+        int x = 1;
+
+        System.out.println(x);
+        x++;
+        System.out.println(x);
+        x++;
+        System.out.println(x);
+        x++;
+        System.out.println(x);
+        x++;
+        System.out.println(x);
+        x++;
+        System.out.println(x);
+        x++;
+        System.out.println(x);
+        x++;
+        System.out.println(x);
+        x++;
+        System.out.println(x);
+        x++;
+        System.out.println(x);
+        x++;
+
+        System.out.println("Numarul panala 1: ");
+        x--;
+        System.out.println(x);
+        x--;
+        System.out.println(x);
+        x--;
+        System.out.println(x);
+        x--;
+        System.out.println(x);
+        x--;
+        System.out.println(x);
+        x--;
+        System.out.println(x);
+        x--;
+        System.out.println(x);
+        x--;
+        System.out.println(x);
+        x--;
+        System.out.println(x);
+        x--;
+        System.out.print(x);
+        x--;
+        System.out.println(x);
+        x--;
+
+
+
+
+
```

Comment:
remove these useless empty lines. One empty line is ok (helps clarity), but two or more empty lines are not recommended because the code is being push at the bottom.